activeactive/skills/skill-restore.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+echo "$green Enter backup dir path:"
+read backupDir
+echo "$green Enter skill id:"
+read skillId
+
+uuidArray=(${skillId//-/ })
+uuid=${uuidArray[0]}
+
+echo -e "$green $(date) Backup started \n"
+
+# Validate dependency module
+# $1 - Name of the dependency module
+# $2 - Command to validate module
+function validate_dependency() {
+  eval $2
+  # Next statement is checking last command success
+  if [ $? -ne 0 ]; then
+    echo -e "$red $(date) Please install ******** $1 ***********  ... Exiting $default"
+    exit 1
+  fi
+}
+
+# Validate required modules exits in target setup
+function validate_setup() {
+  validate_dependency jq "jq --version"
+  echo -e "$(date) Successfully validated required dependencies"
+}
+
+
+function create_tenant_secret() {
+
+  echo -e "$green Starting port forwarding for ai-deployer to create tenant secret \n"
+
+  IDENTITY_SERVER_ENDPOINT=$(kubectl -n uipath get deployment ai-app-deployment -o json | jq -r '.spec.template.spec.containers[0].env[] | select(.name == "IDENTITY_SERVER_ENDPOINT").value')'/connect/token'
+  S2S_CLIENT_ID=$(kubectl -n uipath get secret identity-client-aifabric -o json | jq '.data."Aicenter.Recovery.S2S.ClientId"' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+  S2S_CLIENT_SECRET=$(kubectl -n uipath get secret identity-client-aifabric -o json | jq '.data."Aicenter.Recovery.S2S.ClientSecret"' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+
+  ACCESS_TOKEN=$(curl --location --request POST $IDENTITY_SERVER_ENDPOINT \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id='$S2S_CLIENT_ID \
+  --data-urlencode 'client_secret='$S2S_CLIENT_SECRET \
+  --data-urlencode 'grant_type=client_credentials' \
+  --data-urlencode 'audience=AiFabricRecovery' | jq '.access_token' | sed -e 's/^"//' -e 's/"$//')
+
+  kubectl -n uipath port-forward service/ai-deployer-svc 80:80 --address 0.0.0.0 &
```

Comment:
Externalize the port to some variable and have default value as 80. This way for whatever reason if customer wants to change port number they can only change at one place.